docs/ha-mode.md

```diff
@@ -86,28 +86,36 @@ This domain name, or default "lb-apiserver.kubernetes.local", will be inserted
 into the `/etc/hosts` file of all servers in the `k8s-cluster` group. Note that
 the HAProxy service should as well be HA and requires a VIP management, which
 is out of scope of this doc. Specifying an external LB overrides any internal
-localhost LB configuration.
+localhost LB configuration for masters. Compute nodes will keep using the
+internal LB, when applicable.
 
   Note: In order to achieve HA for HAProxy instances, those must be running on
   the each node in the `k8s-cluster` group as well, but require no VIP, thus
   no VIP management.
 
-Access endpoints are evaluated automagically, as the following:
+Access API endpoints are evaluated automagically, as the following:
 
 | Endpoint type                | kube-master   | non-master          |
 |------------------------------|---------------|---------------------|
-| Local LB (default)           | http://lc:p   | https://lc:nsp      |
+| Local LB (default)           | http://lc:sp  | https://lc:nsp      |
 | External LB, no internal     | https://lb:lp | https://lb:lp       |
-| No ext/int LB                | http://lc:p   | https://m[0].aip:sp |
+| External LB, and internal LB | https://lb:lp | https://lc:nsp      |
+| No ext/int LB, bind 0.0.0.0  | http://lc:sp  | https://m[0].aip:sp |
+| No ext/int LB, a custom bind | http://bip:sp | https://m[0].aip:sp |
```

Comment:
If you are wondering how these apply to the automagic evaluation, here it is:
(given the 1st number represents rows, and the 2nd number stands for columns of that table):

```
kube_apiserver_endpoint: |-
  12/32{% if not is_kube_master and loadbalancer_apiserver_localhost -%}
	   https://localhost: nginx_kube_apiserver_port|default(kube_apiserver_port) 
  51{%- elif is_kube_master and not loadbalancer_apiserver_localhost and not loadbalancer_apiserver is defined and kube_apiserver_bind_address != '0.0.0.0' -%}
	   https:// kube_apiserver_bind_address : kube_apiserver_port 
  11/41{%- elif is_kube_master and not loadbalancer_apiserver is defined -%}
	   https://127.0.0.1: kube_apiserver_port  
  {%- else -%}
  21/22/31{%-   if loadbalancer_apiserver is defined and loadbalancer_apiserver.port is defined -%}
	   https:// apiserver_loadbalancer_domain_name|default('lb-apiserver.kubernetes.local') : loadbalancer_apiserver.port|default(kube_apiserver_port) 
  42/52{%-   else -%}
	   https:// first_kube_master : kube_apiserver_port 
  {%-  endif -%}
  {%- endif %}
```
yeah, it's hard to follow those 10 if..else cases, IKR